test/spec/unit/core/targeting_spec.js

```diff
@@ -309,10 +309,10 @@ describe('targeting tests', function () {
         ['string', '2.3', '2.3'],
         ['number', 2.3, '2.3'],
         ['boolean', true, 'true'],
-        ['string-separated', '2.3,4.5', '2.3, 4.5'],
-        ['array-of-string', ['2.3', '4.5'], '2.3, 4.5'],
-        ['array-of-number', [2.3, 4.5], '2.3, 4.5'],
-        ['array-of-boolean', [true, false], 'true, false']
+        ['string-separated', '2.3,4.5', '2.3,4.5'],
```

Comment:
Maybe have the input still include the space to confirm it trims it afterwards, so to make it instead:
```['string-separated', '2.3, 4.5', '2.3,4.5'],```

Reply:
the `'2.3, 4.5'` returns `'2.3, 4.5'` and this makes some sense since no split/join was done

seems the extra space came to add some readability to the array

Reply:
It now does split since https://github.com/prebid/Prebid.js/commit/3b59a2b310d47092557a5605d7074bcb07a3885a so when pushing through '`2.3, 4.5`' it should now return '`2.3,4.5`' with your changes.

Att he time I was retaining the original behaviour, but maybe https://github.com/prebid/Prebid.js/commit/3b59a2b310d47092557a5605d7074bcb07a3885a should have included a call to `trim()` after splitting...

Reply:
do you mean like on commit `dddcc23` ?

Reply:
> do you mean like on commit https://github.com/prebid/Prebid.js/commit/dddcc23 ?

Yep, exactly what I was thinking.

Reply:
cool!